packages/hatchify-core/src/dataTypes/datetime/getFinalize.ts

```diff
@@ -34,6 +34,13 @@ export function getFinalize(
     control,
     orm: finalizeOrm(props.orm),
 
+    // todo: HATCH-348
+    setClientPropertyValue: () => null,
+    serializeClientPropertyValue: () => null,
+    setClientQueryFilterValue: () => null,
+    serializeClientQueryFilterValue: () => "",
+    setClientPropertyValueFromResponse: () => null,
```

Comment:
@roy-coder 
because of the type changes, these are now required & need to be tested - but the scope of writing tests for this is in a different ticket for frontend (HATCH-348 for dates, HATCH-347 for string/text) - I needed to put empty checks here to satisfy 100% coverage